docs/Overview.md

```diff
@@ -29,8 +36,8 @@ Platformatic DB can expose a SQL database by dynamically mapping it to REST/Open
 and GraphQL endpoints. It supports a limited subset of the SQL query language, but
 also allows developers to add their own custom routes and resolvers.
 
-![Platformatic DB Architecture](./platformatic-db-architecture.png)
-
+<!-- ![Platformatic DB Architecture](./platformatic-db-architecture.png) -->
+a
```

Comment:
I think it would be good to keep the images but if they are being removed, it would be better to delete the line